README.md

```diff
@@ -12,6 +12,7 @@ Make sure that:
 * You have added the public SSH key, often `~/.ssh/id_rsa.pub`, from your machine to [GitHub](https://github.com/settings/keys)
 * You have previously `ssh`-ed into all servers.
   * NOTE: If you are unsure about this, run `bin/check_ssh -e [qa|stage|prod]` and watch the output for any errors!
+* NOTE: if you run `check_cocina`, you may need to ensure that you have the contribsys gem credentials available to for google-books to install the sidekiq-pro gem locally (the credential is already on our deploy target VMs).  You can get the env variable name and value from shared_configs for google-books -- it's in the shared_configs README.
```

Comment:
`gem credentials available to for google-books` looks like it has an extraneous word.